src/components-examples/material/input/input-harness/input-harness-example.html

```diff
@@ -0,0 +1,16 @@
+<mat-form-field>
+  <input matInput placeholder="Favorite food" value="Sushi" name="favorite-food">
```

Comment:
We've been trying to push people towards using `mat-label` instead of `placeholder`.